doc/manual/modules/performance/ref_tuned-profiles-distributed-with-rhel.adoc

```diff
@@ -24,20 +24,36 @@ That is why the `balanced` profile can be generally a better option.
 `throughput-performance`::
 A server profile optimized for high throughput. It disables power savings mechanisms and enables `sysctl` settings that improve the throughput performance of the disk and network IO. CPU governor is set to `performance`.
 
+`accelerator-performance`::
+The `accelerator-performance` profile contains the same tuning as the `throughput-performance` profile. Additionally, it locks the CPU to low C states so that the latency is less than 100us. This improves the performance of certain accelerators, such as GPUs.
+
 `latency-performance`::
 A server profile optimized for low latency. It disables power savings mechanisms and enables `sysctl` settings that improve latency. CPU governor is set to `performance` and the CPU is locked to the low C states (by PM QoS).
++
+It changes the `energy_performance_preference` and `scaling_governor` attribute to the `performance` profile.
 
 `network-latency`::
 A profile for low latency network tuning. It is based on the `latency-performance` profile. It additionally disables transparent huge pages and NUMA balancing, and tunes several other network-related `sysctl` parameters.
++
+It inherits either the `latency-performance` profile which changes the `energy_performance_preference` and `scaling_governor` attribute to the `performance` profile.
```

Comment:
"It inherits either", but there is only one profile.

Reply:
Aha, sorry - fixed now